relay-kafka/src/config.rs

```diff
@@ -102,7 +102,7 @@ pub struct TopicAssignments {
     #[serde(alias = "metrics", alias = "ingest-metrics")]
     pub metrics_sessions: TopicAssignment,
     /// Topic name for all other kinds of metrics. Defaults to the assignment of `metrics`.
-    #[serde(alias = "metrics_transactions", alias = "ingest-generic-metrics")]
+    #[serde(alias = "metrics_transactions", alias = "ingest-performance-metrics")]
```

Comment:
Are we sure it's not both, e.g. single tenant?

Reply:
Yeah, the key here is the logical or default topic, what it gets overridden to in each environment (such as different single tenants) is defined in the `TopicAssignment`.

Reply:
I need to find a way to validate these against what we have registered in https://github.com/getsentry/sentry-kafka-schemas/blob/main/topics/ingest-performance-metrics.yaml to avoid this happening in the future.

Reply:
We're already pulling in `sentry-kafka-schemas`, can probably use it to validate the topics in a test?

Reply:
Sounds good, I need to spend some time to figure how to make this work + also try to remove the duplication of default topics in the file -- will follow up with something shortly.